google-cloud-testing/storage-benchwrapper/pom.xml

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>storage-benchwrapper</artifactId>
+  <version>0.0.0</version>
+  <packaging>jar</packaging>
+  <groupId>com.benchwrapper</groupId>
```

Comment:
I don't think you need a `groupId` here, I'd add a `<parent>` reference and make the artifactId 'google-cloud-benchmarking' or something similar. Take a look at https://github.com/googleapis/google-cloud-java/blob/master/google-cloud-testing/google-cloud-conformance-tests/pom.xml#L6-L14 for an example

Reply:
The `groupId` here should match the Java package and be a domain that we can publish to. We should probably make it `com.google.cloud`.

The java package name should probably live under `com.google.cloud.benchwrapper` unless this is supposed to be a general tool outside of the scope of the cloud libraries.

Reply:
Done